src/server/server.py

```diff
@@ -3693,6 +3695,15 @@ def emit_format_data(**params):
 def emit_pilot_data(**params):
     '''Emits pilot data.'''
     pilots_list = []
+    
+    attrs = []
+    for attr in RHUI.get_pilot_attributes():
+        attrs.append({
+            'name': attr.name,
+            'label': attr.label,
+            'fieldtype': attr.fieldtype
+        })

```

Comment:
Use list comprehensions and consider using something to render objects as JSON-objects instead of hand-coding this here. I don't know what's beste there, one option seems to be https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.json_provider_class, but there is probably smart words written about this in the interwebs somewhere.